compiler/testData/codegen/boxInline/smap/smapWithNewSyntax.kt

```diff
@@ -0,0 +1,35 @@
+// !LANGUAGE: +CorrectSourceMappingSyntax
+// FILE: 1.kt
+package test
+
+public inline fun inlineFun(): String {
```

Comment:
Please make some inline calls inside 1.kt file to generate smap for it (it will allow to check smap parsing during compilation against binaries)

Same for second test

Reply:
ok